README.md

```diff
@@ -7,11 +7,362 @@ possible to create protocol-based bots._
 well-known chat bots like OpenAI's ChatGPT and Anthropic's Claude, but anyone can create
 their own bot by implementing the protocol described in this repository.
 
-## Repo contents
+## Introduction
 
-- [Protocol specification](./docs/spec.md)
-- [Running a bot](./docs/running-a-bot.md)
+This specification provides a way to run custom bots from any web-accessible service
```

Comment:
I think the spec should stay its own file. The README should be introductory, not the full formal spec.